website/docs/r/virtual_machine_scale_set.html.markdown

```diff
@@ -376,6 +376,7 @@ output "principal_id" {
 * `load_balancer_inbound_nat_rules_ids` - (Optional) Specifies an array of references to inbound NAT rules for load balancers.
 * `primary` - (Optional) Specifies if this ip_configuration is the primary one.
 * `public_ip_address_configuration` - (Optional) describes a virtual machines scale set IP Configuration's
+* `application_security_group_ids` - (Optional) Specifies an array of references to application security groups
```

Comment:
can we move this below the second line of documentation for `public_ip_address_configuration`

Reply:
I have moved it above the `public_ip_address_configuration` documentation as it was clearer that it belonged to the ip configuration block.